docs/deploying_csi_vsphere_with_rbac.md

```diff
@@ -117,52 +120,114 @@ Create the secret by running the following command:
-
-##### Deploy `csi-vsphere` as a Pod
+- The YAML to deploy CSI controller assumes that your Kubernetes cluster was deployed using [kubeadm](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/). If you deployed your cluster using alternate means, you will need to modify the either of the YAML files in order to provided necessary files or paths based on your deployment.
 
-The YAML to deploy `csi-vsphere` as a Pod can be found in [vsphere-csi-pod.yaml](https://github.com/kubernetes/cloud-provider-vsphere/raw/master/manifests/csi/vsphere-csi-pod.yaml).
+The YAML to deploy `csi-vsphere-controller` can be found in [vsphere-csi-controller-ss.yaml](https://github.com/kubernetes/cloud-provider-vsphere/raw/master/manifests/csi/vsphere-csi-controller-ss.yaml).
 
 Run the following command:
 
 ```bash
-[k8suser@k8master ~]$ kubectl create -f vsphere-csi-pod.yaml
+[k8suser@k8master ~]$ kubectl create -f vsphere-csi-controller-ss.yaml
 ```
 
-##### Deploy `csi-vsphere` as a DaemonSet
+#### 6. Deploy `csi-vsphere-node`
 
-The YAML to deploy `csi-vsphere` as a DaemonSet can be found in [vsphere-csi-ds.yaml](https://github.com/kubernetes/cloud-provider-vsphere/raw/master/manifests/csi/vsphere-csi-ds.yaml).
+The YAML to deploy `csi-vsphere-node` can be found in [vsphere-csi-node-ds.yaml](https://github.com/kubernetes/cloud-provider-vsphere/raw/master/manifests/csi/vsphere-csi-node-ds.yaml).
 
 Run the following command:
 
 ```bash
-[k8suser@k8master ~]$ kubectl create -f vsphere-csi-ds.yaml
+[k8suser@k8master ~]$ kubectl create -f vsphere-csi-node-ds.yaml
+```
+
+#### 7. Create CRDs for CSI driver registration
+
+You can find the CRDs to register the CSI nodes in [vsphere-csi-crd.yaml](https://github.com/kubernetes/cloud-provider-vsphere/raw/master/manifests/csi/vsphere-csi-crd.yaml).
+
+To apply them to your Kubernetes cluster, run the following command:
+
+```bash
+[k8suser@k8master ~]$ kubectl create -f vsphere-csi-crd.yaml
+```
+
+#### 8. Create your StorageClass and PersistentVolumeClaim by Example
+
+Each StorageClass (SC) is going to be unique to each user as it depends on the vSphere configuration you have. The PersistentVolumeClaim (PVC) is also therefore unique since the PVC depends on the SC. You can find examples of each in the [manifests/csi](https://github.com/kubernetes/cloud-provider-vsphere/tree/master/manifests/csi) directory for reference. The important thing to note in the [StorageClass](https://github.com/kubernetes/cloud-provider-vsphere/tree/master/manifests/csi/example-vsphere-sc.yaml) as seen below is that you need to provide as paramters the type of datastore you will be using (`DatastoreCluster` or `Datastore`) and it's corresponding name.
```

Comment:
I'm not sure I agree the SC is unique per user. Each SC should define a specific datastore or datastore cluster that is accessible via the credentials given to the CSI plugin. But that doesn't mean that SC can't be used by multiple users, does it?

Reply:
When I mean user... I mean the vsphere/k8s admin. They are unique because every "user" will have a unique names for their own datastores.